docs/pbc/all/payment/unzer/extend-and-customize/frontend-for-a-credit-card.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Don't need a separate doc just to give a link. Shall we move it to `unzer.md`?

Reply:
You can create a next steps sections in that doc and include this link there as well.